Sources/MagicSDK/Core/Relayer/Types/ProductAnnouncement.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ProductAnnouncement.swift
+//
+//
+//  Created by Arian Flores - Magic on 2/27/24.
+//
+
+import Foundation
+
+struct AnnouncementResponse: Codable {
```

Comment:
This is redundant with Web3Response class

Reply:
I agree, I tried making it work, but ran into trouble with the fact that `id` can sometimes be `null`. I'll make the change and show you what I mean.

Reply:
I'll also add that the payloadStr from Relayer looks like the following:

```
{
  "msgType": "MAGIC_SEND_PRODUCT_ANNOUNCEMENT-eyJBUElfS0VZIjoicGtfbGl2ZV9CRTI1NDc4RDUxMjkzQkM3IiwiaG9zdCI6Imh0dHA6Ly8xOTIuMTY4LjEuMjE5OjMwMTYvIiwibG9jYWxlIjoiZW5fVVMiLCJidW5kbGVJZCI6ImxpbmsubWFnaWMubWFnaWMtc3BtLWlvcy1kZW1vIiwiRVRIX05FVFdPUksiOiJtYWlubmV0Iiwic2RrIjoibWFnaWMtc2RrLWlvcyJ9",
  "response": {
    "jsonrpc": "2.0",
    "id": null,
    "result": {
      "product_announcement": "`Universal Wallet API Keys` will lose support on all of Magic's SDKs. Use a `Dedicated Wallet API Key` instead to prevent disruption of the wallet service. Visit https://magic.link/posts/unifyingwaas for more information."
    }
  }
}

```

Which does not resemble a [Web3Response](https://github.com/magiclabs/Web3.swift/blob/master/Sources/Core/Providers/Web3Provider.swift#L18-L69) struct.